pkg/test/framework/components/echo/check/checkers.go

```diff
@@ -249,6 +250,16 @@ func Hostname(expected string) echo.Checker {
 	})
 }
 
+// ResponseHosts checks that the response hostnames match the expected values.
```

Comment:
nit: 
```suggestion
// ResponseHosts checks that the response hostname matches an expected value.
```

Reply:
This func is specific for multiple hostnames, the `Hostname` func above is for single hostname check.